templates/permission/index.html.twig

```diff
@@ -37,9 +37,14 @@
                     <label>Username</label>
                     <input type="text" class="form-control" name="P_username" required="required" />
 
-                    <label>Roles</label>
-                    <input type="text" class="form-control" name="P_roles" />
-
+                    {% if app.space_roles %}
```

Comment:
@kaushikindianic we need to make sure app.space_roles is there otherwise it will crash